### PR TITLE
Update Rutgers faculty

### PR DESCRIPTION
**Pull request by zlangley**

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -14836,7 +14837,7 @@ Per Wahlund,Uppsala University,https://katalog.uu.se/empinfo/?id=XX2255,NOSCHOLA
 Peraphon Sophatsathit,Chulalongkorn University,https://cop.car.chula.ac.th/%E0%B8%9E%E0%B8%B5%E0%B8%A3%E0%B8%B0%E0%B8%9E%E0%B8%B1%E0%B8%99%E0%B8%98%E0%B9%8C,NOSCHOLARPAGE
 Perdita Stevens,University of Edinburgh,http://homepages.inf.ed.ac.uk/perdita,KdBhbbYAAAAJ
 Pere Barlet-Ros,Polytechnic University of Catalonia,http://people.ac.upc.edu/pbarlet,NOSCHOLARPAGE
-Periklis A. Papakonstantinou,Rutgers University,https://www.cs.rutgers.edu/people/faculty/affiliated-faculty,NOSCHOLARPAGE
+Periklis A. Papakonstantinou,Rutgers University,https://www.cs.rutgers.edu/people/faculty/affiliated-faculty,x9wgKOkAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not a personal home page.

File: csrankings.csv [hunk 2]
```diff
@@ -20169,7 +20171,7 @@ William Robson Schwartz,UFMG,http://homepages.dcc.ufmg.br/~william,koXKq5EAAAAJ
 William S. Cleveland,Purdue University,http://www.stat.purdue.edu/~wsc,ds52UHcAAAAJ
 William S. Evans,University of British Columbia,https://www.cs.ubc.ca/people/william-evans,JU1Z-DcAAAAJ
 William Shoaff,Florida Institute of Technology,http://fit.academia.edu/WilliamShoaff,NOSCHOLARPAGE
-William Steiger,Rutgers University,http://www.cs.rutgers.edu/~steiger,NOSCHOLARPAGE
+William Steiger,Rutgers University,http://www.cs.rutgers.edu/~steiger,8rnvCfgAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Needs a `dblp-aliases.csv` entry.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by zlangley** [comment]
Updated.

File: csrankings.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings.csv]: The URL doesn't point to the faculty member's home page.

**Review by emeryberger — APPROVED** [review]

**Pull request merged**